blog.html

```diff
@@ -22,6 +22,46 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_360_release_announcement"></a>
+                    <a href="#apache_kafka_360_release_announcement">Apache Kafka 3.6.0 Release Announcement</a>
+                </h2>
+                15 Sep 2023 - Satish Duggana (<a href="https://twitter.com/0xeed">@SatishDuggana</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.6.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>The ability to migrate Kafka clusters from ZK to KRaft mode with no downtime is still an early access feature. It is currently only suitable for testing in non production environments. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-866+ZooKeeper+to+KRaft+Migration">KIP-866</a> for more details.</p>
```

Comment:
Should we add a note about support of delegation token for kraft? [1] 
Suggesting this because it takes us one step closer to completing the missing features for kraft https://kafka.apache.org/documentation.html#kraft_missing


[1] https://issues.apache.org/jira/browse/KAFKA-15219

Reply:
@mumrah is covering ZK to KRaft migration documentation in https://github.com/apache/kafka-site/pull/545/.

Reply:
Correct me I am wrong but delegation token support is different from Zk to KRaft migration.

Reply:
I agree, @divijvaidya, we should call out KRaft support for delegation tokens. How about a short blurb after the ZK migration paragraph:

```
<p>Support for Delegation Tokens in KRaft (<a href="https://issues.apache.org/jira/browse/KAFKA-15219">KAFKA-15219</a>) was completed in 3.6, further reducing the gap of features between ZooKeeper-based Kafka clusters and KRaft. Migration of delegation tokens from ZooKeeper to KRaft is also included in 3.6.</p>
```

cc @pprovenzano

Reply:
I meant delegation token support as part of KRaft related updates, including the migration. Added the blurb as suggested.